### PR TITLE
feat: upgrades ganache to 7.5.0

### PR DESCRIPTION
**Pull request by jeffsmale90**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -126,8 +127,6 @@
     "eslint-plugin-prettier": "^3.1.0",
     "eslint-plugin-react": "^7.14.3",
     "eslint-plugin-react-hooks": "^2.1.0",
-    "ganache": "7.0.0-filecoin.6",
-    "ganache-core": "2.13.1",
```
[the remark this answers is not in the document]
Comment (jeffsmale90) [reply package.json:130]: Good call.

We now will use ganache-core (Ganache v2) for legacy (Ethereum) workspaces _only_. Ganache v7 will be used on all new (Ethereum) workspaces (and Quickstart)

File: src/integrations/ethereum/renderer/screens/contracts/ContractDetails.js [hunk 2]
```diff
@@ -75,29 +75,29 @@ class ContractDetails extends Component {
...
     );
 
     const body = this.renderIssueBody(
-      sanitizeError(this.props.workspaces.current.shownContract.state.error),
+      sanitizeError(this.props.workspaces.current.shownContract.state.error)
     );
 
     shell.openExternal(
-      `https://github.com/trufflesuite/ganache/issues/new?title=${title}&body=${body}`,
+      `https://github.com/trufflesuite/ganache-ui/issues/new?title=${title}&body=${body}`
```
Comment (jeffsmale90) [inline src/integrations/ethereum/renderer/screens/contracts/ContractDetails.js:86]: This is the only actual change in this file (updating the link to point to the correct project)

File: src/main/index.js [hunk 3]
```diff
@@ -533,25 +580,22 @@ app.on('ready', async () => {
...
+    if (workspace) {
+      const globalSettings = global.getAll();
+      const workspaceSettings = workspace.settings.getAll();
+      mainWindow.webContents.send(
+        SET_SERVER_STARTED,
+        globalSettings,
+        workspaceSettings,
+        startupMode
+      );
     }
```
Comment (jeffsmale90) [inline src/main/index.js:599]: This is the key change in this file

File: package.json [hunk 4]
```diff
@@ -5,7 +5,7 @@
   "description": "Personal Blockchain for Ethereum",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/trufflesuite/ganache-ui.git"
+    "url": "git+https://github.com/trufflesuite/ganache.git"
```
Comment (davidmurdoch) [inline package.json:8]: I'm guessing these changes aren't meant to be in here?

Comment (jeffsmale90) [reply package.json:8]: I've reverted the github link fixes and [put them into a separate PR](https://github.com/trufflesuite/ganache-ui/pull/5123/files). Figured this is already a little messy with all of the style changes from prettier, including more unrelated changes wasn't going to make it easier to follow.

**Review by davidmurdoch — APPROVED** [review]
Too many formatting changes to really review this. I hope it works. :-p

File: static/node/package.json [hunk 5]
```diff
@@ -1,19 +1,18 @@
...
-        "web3-providers-ws": "1.2.6"
-    }
+  "dependencies": {
+    "@ganache/filecoin": "0.1.5",
+    "@truffle/config": "1.2.17",
+    "@truffle/decoder": "3.0.16",
+    "electron": "12.0.2",
+    "electron-fetch": "1.7.3",
+    "fs-extra": "9.0.0",
+    "ganache": "^7.5.0",
```
Comment (davidmurdoch) [inline static/node/package.json:9]: Should `ganache-core` be in this package.json still?

Comment (jeffsmale90) [reply static/node/package.json:9]: Yes! Should probs also pin the "ganache" dependency.

**Pull request merged**